### PR TITLE
filter get eventsByMovie by theatre

### PR DESCRIPTION
**Pull request by cll1n**

### CODE REVIEW

**Review by Hajny25 — APPROVED** [review]

File: src/repositories/event_repository_test.go [hunk 1]
```diff
@@ -344,7 +345,7 @@ func TestGetEventsForMovie(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			db, mock, err := sqlmock.New(sqlmock.QueryMatcherOption(sqlmock.QueryMatcherRegexp))
+			db, mock, err := sqlmock.New()
```
Comment (Kieberts) [inline src/repositories/event_repository_test.go:348]: Why did you remove the regex?

**Review by Kieberts — APPROVED** [review]
👍🏼

File: src/repositories/event_repository_test.go (hunk 1 above)
Comment (Hajny25) [reply src/repositories/event_repository_test.go:348]: its the default behaviour
